boards/riscv/rv32m1_vega/pinmux.c

```diff
@@ -82,6 +77,18 @@ static int rv32m1_vega_pinmux_init(struct device *dev)
 	pinmux_pin_set(portb, 22, PORT_PCR_MUX(kPORT_MuxAlt2));
 #endif
 
+#ifdef DT_OPENISA_RV32M1_TPM_PWM_2_BASE_ADDRESS
```

Comment:
This is always going to be defined. I think we should use `CONFIG_PWM_2` instead

Reply:
No, it will only be defined if the given device has `status = "okay"` in the dts. As I understand it, the plan is to move away from enabling instances in Kconfig. I discussed this with @galak at ELCE 2019, and it is similar to what we are doing in `boards/arm/twr_ke18f/pinmux.c` for the ADC12.

Reply:
Given #21803, can we change this to `CONFIG_PWM_2` for now? Otherwise `samples/basic/blinky` will be broken